src/data-workspace/print-area/printable-area.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types'
+import React, { useEffect } from 'react'
+import { usePrintableArea } from './print-area-context.js'
+
+const PrintableArea = ({ children }) => {
```

Comment:
this `PrintableArea` wraps any components we would like to print. It's needed for two reasons:

- to set the area to print in the Context, so that it's accessible from a different part of the app tree (the options menu)
- to wrap the component in a DOM element and set a `ref` to it. The `react-to-print` requires a `ref` to the DOM element to print and it's not possible to set a ref on functional components.

Reply:
Solving it this way makes sense to me 👍

Reply:
Solving it this way makes sense to me 👍